### PR TITLE
Fix typescript-eslint-jest example to remove Prettier package causing errors

### PR DESCRIPTION
**Pull request by rsipakov**
Delete "prettier/@typescript-eslint" from .eslintrc.json. 

After update to the last version I getting this error on running lint: ESLint couldn't find the config "prettier/@typescript-eslint" to extend from. 
I explore this issue https://github.com/prettier/eslint-config-prettier/pull/175 and I think those extensions aren't needed anymore.

### CODE REVIEW

File: examples/with-typescript-eslint-jest/.eslintrc.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     // Uncomment the following lines to enable eslint-config-prettier
     // Is not enabled right now to avoid issues with the Next.js repo
     // "prettier",
-    // "prettier/@typescript-eslint"
+    // "prettier/@typescript-eslint" // delete according to this information https://github.com/prettier/eslint-config-prettier/pull/175
```
Comment (leerob) [inline examples/with-typescript-eslint-jest/.eslintrc.json:11]: Wouldn't we just delete this line?

Comment (rsipakov) [reply examples/with-typescript-eslint-jest/.eslintrc.json:11]: Yes, I think this is the best option.

Comment (leerob) [inline examples/with-typescript-eslint-jest/.eslintrc.json:11]:
```suggestion
```

**Review by leerob — APPROVED** [review]
Works for me. If you end up figuring out a proper solution for this, we'd love to have it back here. But hopefully this unblocks people in a broken state on this example.

thank you!

**Pull request merged**